data_managers/data_manager_hisat_index_builder/tool_dependencies.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<tool_dependency>
+  <package name="hisat" version="0.1.5">
+      <repository name="package_hisat_0_1_5" owner="iuc" prior_installation_required="False" />
```

Comment:
`s/0.1.5/0.1.6/` and `s/package_hisat_0_1_5/package_hisat_0_1_6/`

This file is not properly indented.